docs/manual/docs/about/comparison-powershell.md

```diff
@@ -47,7 +47,7 @@ Get-SPOMigrationJobProgress|Microsoft.Online.SharePoint.PowerShell|
 Get-SPOMigrationJobStatus|Microsoft.Online.SharePoint.PowerShell|
 Get-SPOMultiGeoCompanyAllowedDataLocation|Microsoft.Online.SharePoint.PowerShell|
 Get-SPOMultiGeoExperience|Microsoft.Online.SharePoint.PowerShell|
-Get-SPOOrgAssetsLibrary|Microsoft.Online.SharePoint.PowerShell|
+Get-SPOOrgAssetsLibrary|Microsoft.Online.SharePoint.PowerShell|[spo orgassetslibrary list](../cmd/spo/orgassetslibrary/orgassetslibrary-kust.md)
```

Comment:
Fixed a typo in the link

Reply:
Pretty weird as I had tested without the X-req and it did work on my side as well.. So thanks for fixing it :)